pkg/controller/model_binding_pair.go

```diff
@@ -345,6 +385,20 @@ func buildModelBindingPair(mbPair *types.ModelBindingPair) *types.ModelBindingPa
 					}
 				}
 			}
+
+			genericComponents := mbPair.Model.Spec.GenericComponents
```

Comment:
A comment about what this block of code is doing may be helpful

Reply:
There is a comment at the outer for loop where we mention that we are processing rest connections.  Do you have something in mind?

Reply:
maybe something like "Adding environment variables for generic components"

I just expanded the code and I see there are similar blocks for each component type.  So maybe ok as is or add a single line comment to each block.

Reply:
Sounds reasonable.  I will add to each block.

Reply:
Fixed